apps/mempool/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  web:
+    image: mempool/frontend:v2.1.0@sha256:ee952d10f6463bff2de21bcceac8ff8f43ba4a5bba2a1aa39901c8a05761a9b9
+    user: "1000:1000"
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 1m
+    command: "./wait-for db:3306 --timeout=720 -- nginx -g 'daemon off;'"
+    ports:
+      - ${APP_MEMPOOL_PORT}:8080
+    environment:
+      BACKEND_MAINNET_HTTP_HOST: $APP_MEMPOOL_API_IP
+    networks:
+      default:
+        ipv4_address: $APP_MEMPOOL_IP
+  api:
+    image: mempool/backend:v2.1.0@sha256:237270551748a3445ae84d1e4e390ed1610486725bd7e8b561e3b1e10155f575
+    user: "1000:1000"
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 1m
+    command: "./wait-for-it.sh db:3306 --timeout=720 --strict -- ./start.sh"
+    volumes:
+      - ${APP_DATA_DIR}/data:/backend/cache    
+    environment:
+      RPC_HOST: $BITCOIN_IP
+      RPC_PORT: $BITCOIN_RPC_PORT
+      RPC_USER: $BITCOIN_RPC_USER
+      RPC_PASS: $BITCOIN_RPC_PASS
+      ELECTRS_HOST: $ELECTRUM_IP
+      ELECTRS_PORT: $ELECTRUM_PORT
+      MYSQL_HOST: $APP_MEMPOOL_DB_IP
+      CACHE_DIR: "/backend/cache/"
+    networks:
+       default:
+         ipv4_address: $APP_MEMPOOL_API_IP
+  db:
+    image: mariadb:10.5.8
+    user: "1000:1000"
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/mysql/data:/var/lib/mysql
+      - ${APP_DATA_DIR}/mysql/db-scripts:/docker-entrypoint-initdb.d
+    environment:
+      MYSQL_DATABASE: "mempool"
+      MYSQL_USER: "mempool"
+      MYSQL_PASSWORD: "mempool"
+      MYSQL_ROOT_PASSWORD: "moneyprintergobrrr"
```

Comment:
How is this password passed into the API container?

Reply:
If you're just implicitly relying on the value being the same in the mempool app, it would be better to instead explicitly pass it in.

Otherwise if the default password value is ever changed in the mempool app things will suddenly break when we update and it won't be obvious why to the person implementing the update if they aren't familiar with the currently implicit password behaviour.

With the app configurations, it's always better to favour being more explicit at the cost of extra lines of code.